src/legacy/core_plugins/kbn_vislib_vis_types/public/controls/point_series/value_axes.html

```diff
@@ -255,7 +255,33 @@
               i18n-default-message="Scale to Data Bounds"
             ></label>
             <div class="visEditorSidebar__formControl">
-              <input class="kuiCheckBox" id="{ 'defaultYExtents' + $index }" type="checkbox" ng-model="axis.scale.defaultYExtents">
+              <input class="kuiCheckBox" id="{ 'defaultYExtents' + $index }" type="checkbox" ng-model="axis.scale.defaultYExtents" ng-change="updateBottomMargin(axis)">
+            </div>
+          </div>
+
+          <div ng-if="axis.scale.defaultYExtents">
+            <div class="visEditorSidebar__formRow">
+              <label
+                class="visEditorSidebar__formLabel"
+                i18n-id="kbnVislibVisTypes.controls.pointSeries.valueAxes.scaleToDataBounds.bottomMargin"
+                i18n-default-message="Bottom Margin"
+              ></label>
+              <div class="visEditorSidebar__formControl">
+                <input
+                  name="yBottomMargin"
+                  class="kuiInput visEditorSidebar__input"
+                  type="number"
+                  step="0.1"
+                  greater-or-equal-than="0"
```

Comment:
If we constraint here to only positive value, we should also configure how to handle the case of a visualization with all negative values